upgradepg16 (#119233)
Azure/azure-content: articles/postgresql/flexible-server/concepts-major-version-upgrade.md

-	Servers configured with logical replication slots aren't supported. 

> - In-place major version upgrade doesn't support upgrading to version 16, our team is actively working on this feature and it will be made available in the near future.

Reviewer: ```suggestion
- The in-place major version upgrade doesn't yet support an upgrade to version 16.
```
@kabharati - Future promises are [not allowed](https://review.learn.microsoft.com/en-us/help/contribute/contribute-get-started-channel-guidance?branch=main). Revisions OK? If you accept, I will commit this change and merge the PR.
